triagebot.toml

```diff
@@ -102,6 +102,15 @@ message_on_add = """\
 """
 message_on_remove = "Issue #{number}'s prioritization request has been removed."
 
+[notify-zulip."T-rustdoc"]
+required_labels = ["I-nominated"]
+zulip_stream = 266220 # #rustdoc
+topic = "nominated: #{number}"
+message_on_add = """\
+@*T-rustdoc* issue #{number} "{title}" has been nominated for `T-rustdoc` discussion.
+"""
+message_on_remove = "Issue #{number}'s nomination request has been removed."
```

Comment:
The one issue with this setup is that we will only be notified if *`T-rustdoc`* is removed—nothing will happen if `I-nominated` is removed. It might be worth getting rid of the `message_on_remove` since it will be kind of confusing.

Reply:
Does an array work here? cc @Mark-Simulacrum can we do `[[notify-zulip."I-nominated"]]` here?

Reply:
No, I don't think so, but I also don't really know what you mean by that. That said, it seems quite reasonable for triagebot to detect the removal of a required label and treat that just like removal of T-rustdoc; I think the reasonable treatment is that the whole set of labels must be present and removal of any leads to a notification of such. Happy to accept a PR to that effect.

Reply:
> That said, it seems quite reasonable for triagebot to detect the removal of a required label and treat that just like removal of T-rustdoc; I think the reasonable treatment is that the whole set of labels must be present and removal of any leads to a notification of such. Happy to accept a PR to that effect.

That might be a natural extension of the work in rust-lang/triagebot#1078.

Reply:
> No, I don't think so, but I also don't really know what you mean by that.

I think he means that we would be notified as soon as both `I-nominated` and `T-rustdoc` were present, and we would be notified again as soon as one of them was removed. Treating the removal of a required label the same as the removal of the main label would have the same effect, but gives the labels an asymmetric relationship (`T-rustdoc` is considered the "main" label by triagebot).